apps/api/src/app/integrations/usecases/get-novu-integration/get-novu-integration.usecase.ts

```diff
@@ -19,7 +19,7 @@ export class GetNovuIntegration {
   ) {}
 
   async execute(command: GetNovuIntegrationCommand): Promise<IntegrationEntity | undefined> {
-    if (!process.env.NOVU_EMAIL_INTEGRATION_API_KEY) {
+    if (!process.env.NOVU_EMAIL_INTEGRATION_API_KEY || !command.channelType) {
```

Comment:
it shouldn't be possible that there is no `command.channelType` unless it's null somehow... if it is, then please update the types

Reply:
thanks for the clarification, let's leave it here, but please update the type of the `channelType` to be optional in the `GetNovuIntegrationCommand`